package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "nfl-api",
+  "version": "1.0.0",
+  "description": "## Instructions",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "./node_modules/.bin/nodemon index.js"
```

Comment:
Note from a crazy person: I like to keep these alphabetized to make my life easier when I have 15 of them.